src/lib/Api/api.ts

```diff
@@ -153,15 +155,18 @@ export default class Api {
    * @param {string} opts.contractAddress address of the contract to get the list of NFTs
    * @returns {Promise<object>} List of NFTs with metadata
    */
-  async getNFTsForCollection(opts: ContractAddressOptions): Promise<NftDTO> {
+  async getNFTsForCollection(opts: GetNftsForCollectionOptions): Promise<NftDTO> {
     if (!opts.contractAddress || !utils.isAddress(opts.contractAddress)) {
       log.throwMissingArgumentError(Logger.message.invalid_contract_address, {
         location: Logger.location.SDK_GETNFTSFORCOLLECTION,
       });
     }
     const apiUrl = `${this.apiPath}/nfts/${opts.contractAddress}/tokens`;
 
-    const { data } = await this.httpClient.get(apiUrl, { cursor: opts.cursor });
+    const { data } = await this.httpClient.get(apiUrl, {
+      cursor: opts.cursor,
+      resync: opts.resync,
```

Comment:
what about this?
`resync: !!opts.resync`
so it will be added only when not null/false